almaoai_harvest.py

```diff
@@ -42,7 +43,12 @@ def almaoai_harvest(ds, **kwargs):
         deletedfile = open(deletedfilename, 'w')
 
         sickle = Sickle(endpoint_url)
-        records = sickle.ListRecords(**{'metadataPrefix': 'marc21', 'set': 'blacklight', 'from': '{}'.format(date)})
+        try:
+            records = sickle.ListRecords(**{'metadataPrefix': 'marc21', 'set': 'blacklight', 'from': '{}'.format(date)})
+        except :
```

Comment:
is this meant to be except any error as no records present? or does this library & the ListRecords method limit the exception possibilities here?

Reply:
Question here is not a blocker, but I am curious. Note: outdated is because I read via commit to understand the work flow.